template-only-docs/application-requirements.md

```diff
@@ -26,7 +26,7 @@ The infra template includes an example "hello, world" application that works wit
 
 ## Template Applications
 
-You can the following template applications with the template infrastructure
+You can use the following template applications with the template infrastructure. Each of these includes a script to generate a repository based on the template, which should be run from [the `/app` directory](/app).
```

Comment:
I don't think the last part of the sentence is accurate and can probably be removed, for example in the nextjs template, the installation script is meant to be run from the project root
<img width="880" alt="image" src="https://github.com/navapbc/template-infra/assets/447859/f34faaff-dda8-42b2-ad42-9b1441f742c3">

no strong feelings about the wording but maybe something like this
```suggestion
You can use the following template applications with the template infrastructure. Each of these includes a script to generate a working application that works with this infra template.
```